boot/bootutil/include/bootutil/crypto/rsa.h

```diff
@@ -0,0 +1,356 @@
+/*

```

Comment:
This entire file has DOS line endings. Please fix the commits that add it rather than a new commit.